acquisitions/src/main/resources/thunderjet/mod-orders/features/create-open-composite-order.feature

```diff
@@ -0,0 +1,157 @@
+  # MODORDERS-1098
+  Feature: Create open composite order
+
+    Background:
+      * print karate.info.scenarioName
+
+      * url baseUrl
+
+      * call login testAdmin
+      * def okapitokenAdmin = okapitoken
+      * call login testUser
+      * def okapitokenUser = okapitoken
+
+      * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json' }
+      * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json' }
+
+      * configure headers = headersUser
+      * call variables
+
+
+    Scenario: Create open composite order
```

Comment:
it is better to reanme scenario name by considering all opearings inside of scenario. 
More like Create,.... Verify....

Reply:
OK, renamed "Create open composite order and check results".